packages/wonder-blocks-birthday-picker/src/components/birthday-picker.tsx

```diff
@@ -306,6 +306,7 @@ export default class BirthdayPicker extends React.Component<Props, State> {
         const minWidth = this.getMonthYearWidth(monthYearOnly);
         return (
             <SingleSelect
+                aria-label={this.labels.month}
```

Comment:
Nice improvements! 

Do you know what the labelling behaviour is like when the BirthdayPicker is wrapped by a `<legend>` element in a `<fieldset>`? I'm wondering because the [BirthdayPicker wrapper in webapp](https://github.com/Khan/webapp/blob/9f1ec10df5fd724cc77e601b04c444a062dfba08/services/static/javascript/components/birthday-picker/birthday-picker.tsx#L106-L107)       uses fieldset + legend to provide a visual label to the birthday picker too (since it's a group of inputs)!

Reply:
Good question! It's kind of a description for the group more than a label, since the dropdowns aren't actually labeled with that legend content. The accessible name for each field starting with this PR is "month", "day", or "year", and the legend describes them all.